### ISSUE TITLE
Error messages being stripped out of their codes.

### ISSUE THREAD

**Issue by rarvolt** [apragacz/django-rest-registration#160]
### Checklist

*   [x] I read [Contribution Guidelines](https://github.com/apragacz/django-rest-registration/blob/master/CONTRIBUTING.md#issues)
*   [x] I searched existing issues before opening this one
*   [x] This is not a major security issue
*   [x] I reproduced the bug with the newest version

### Describe the bug
I wanted to send error codes instead of error messages to the front so it could handle error logic there based on codes, but in the function [`_validate_user_password`](https://github.com/apragacz/django-rest-registration/blob/master/rest_registration/utils/validation.py#L64) the error messages are stripped out of their codes and instead, they have an original message and a default `invalid` code.

I want to override DRFs `exception_handler` with a function like this:
```python
from typing import Optional

from rest_framework.exceptions import ValidationError
from rest_framework.response import Response
from rest_framework.views import exception_handler


def code_exception_handler(exc: Exception, context) -> Optional[Response]:
    response = exception_handler(exc, context)

    if response is not None and isinstance(exc, ValidationError):
        codes = exc.get_codes()
        if isinstance(codes, dict):
            response.data = codes
        else:
            if not isinstance(codes, list):
                codes = [codes]
            response.data = {
                'non_field_errors': codes
            }

    return response
```
### Expected behavior
In this case registration:
```json
{
  "password": ["password_too_common", "password_too_short", "password_entirely_numeric"],
  "non_field_errors": []
}
```
### Actual behavior
```json
{
  "password": ["invalid", "invalid", "invalid"],
  "non_field_errors": []
}
```

Was this intentional and is there a reason behind stripping messages of their codes?

**Comment by apragacz** [comment]
Hi @rarvolt ,
this was an lazy attempt to extract any information from the original Django ValidationError. I already have improvement which should also transfer the error codes - will share that one soon (this week).

**Comment by apragacz** [comment]
@rarvolt this issue should be addressed in version `0.6.4`. Let me know if it works as expected.

**Comment by rarvolt** [comment]
@apragacz works great, thanks.
